packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1488,7 +1488,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             };
             // back-compat 0.59.3000 - Older document may either write summaryCount or not write it at all. If it does
```

Comment:
Update this comment, don't need the backcompat comment.

Probably put a disclaimer that this was renamed from summaryCount, for when someone sees summaryCount in telemetry (on an old doc) and searches for it.  And maybe... try to succinctly explain that the count would get reset for docs that weren't opened for several months?